### PR TITLE
Proof dumb component

### PR DESCRIPTION
**Pull request by chrisnojima**
@keybase/react-hackers dumb proofs page

### CODE REVIEW

File: react-native/react/tracker/proofs-render.desktop.js [hunk 1]
```diff
@@ -4,32 +4,76 @@
...
       </div>
     )
   }
 }
+
+ProofsRender.propTypes = {
+  platformProofs: React.PropTypes.arrayOf(React.PropTypes.shape({
+    platform: React.PropTypes.shape({
+      icon: React.PropTypes.string.isRequired,
+      name: React.PropTypes.string,
```
Comment (MarcoPolo) [inline react-native/react/tracker/proofs-render.desktop.js]: is this not required?

File: react-native/react/tracker/proofs-render.desktop.js [hunk 2]
```diff
@@ -4,32 +4,76 @@
...
+      username: React.PropTypes.string,
+      uri: React.PropTypes.string
+    }).isRequired,
+    proof: React.PropTypes.shape({
+      title: React.PropTypes.string,
+      uri: React.PropTypes.string,
+      status: React.PropTypes.oneOf([verified, checking, deleted, unreachable, pending]).isRequired,
+      meta: React.PropTypes.string
```
Comment (MarcoPolo) [inline react-native/react/tracker/proofs-render.desktop.js]: only `status` is required?

**Comment by MarcoPolo** [comment]
LG%C

File: react-native/react/tracker/proofs-render.desktop.js (hunk 1 above)
Comment (chrisnojima) [inline react-native/react/tracker/proofs-render.desktop.js]: if its a website the name isn't really a thing, other than the uri. we could say it has to be duped there...

File: react-native/react/tracker/proofs-render.desktop.js (hunk 2 above)
Comment (chrisnojima) [inline react-native/react/tracker/proofs-render.desktop.js]: yeah, meta is only things like 'new' or 'updated' or something. most of the time its null actually

File: react-native/react/tracker/proofs-render.desktop.js (hunk 1 above)
Comment (MarcoPolo) [inline react-native/react/tracker/proofs-render.desktop.js]: if it's a website the name is `"web"`, right? it is still necessary

**Comment by Sidnicious** [comment]
:+1:

**Pull request merged**